modules/runner/src/integrationTest/java/org/apache/ignite/internal/table/ItTableScanTest.java

```diff
@@ -768,4 +789,26 @@ private static Row createKeyRow(int id) {
 
         return new Row(SCHEMA, new ByteBufferRow(rowBuilder.toBytes()));
     }
+
+    /**
+     * Starts an RW transaction and enlists the specified partition in it.
+     *
+     * @param partId Partition ID.
+     * @return Transaction.
+     */
+    private InternalTransaction startTxWithEnlistedPartition(int partId) {
+        Ignite ignite = CLUSTER_NODES.get(0);
+
+        InternalTransaction tx = (InternalTransaction) ignite.transactions().begin();
+
+        InternalTable table = ((TableImpl) ignite.tables().table(TABLE_NAME)).internalTable();
+        TablePartitionId tblPartId = new TablePartitionId(table.tableId(), partId);
+        RaftGroupService raftSvc = table.partitionRaftGroupService(partId);
+        long term = raftSvc.refreshAndGetLeaderWithTerm().join().term();
```

Comment:
can we amend join() to IgniteTestUtils#await ?

Reply:
No, since `IgniteTestUtils` should be used only in tests.

Reply:
oops... sorry

Fixed in both tests, thanks.